src/psyclone/psyir/transformations/omp_taskwait_trans.py

```diff
@@ -67,7 +67,8 @@ class OMPTaskwaitTrans(Transformation):
     >>> psy = PSyFactory(api).create(invokeInfo)
     >>>
     >>> from psyclone.transformations import OMPParallelTrans, OMPSingleTrans
-    >>> from psyclone.transformations import OMPTaskloopTrans, OMPTaskwaitTrans
+    >>> from psyclone.transformations import OMPTaskloopTrans
+    >>> from psyclone.psyir.transformations import OMPTaskwaitTrans
     >>> singletrans = OMPSingleTrans()
```

Comment:
This same mistake is also made in the docstring of OMPTaskLoopTrans. Please could you fix it there too.

Reply:
Done :)